flask/app/datasets.py

```diff
@@ -120,13 +122,43 @@ def list_datasets(page: int, limit: int) -> Response:
         filters.append(models.Dataset.dataset_id == dataset_id)
 
     participant_codename = request.args.get("participant_codename", type=str)
+    participant_codename_exact_match = request.args.get(
+        "participant_codename_exact_match", type=str_to_bool, default=False
+    )
```

Comment:
I think instead of the str_to_bool thing you might be able to simply check for the existence of the query string parameter and use that.

Reply:
Didn't think about that! I thought it would be more explicit this way but I'm open to changing it